ntp-proto/src/filter.rs

```diff
@@ -9,7 +9,7 @@
 
 use crate::{packet::NtpLeapIndicator, NtpDuration, NtpHeader, NtpTimestamp};
 
-const MAX_STRATUM: u8 = 16;
+pub const MAX_STRATUM: u8 = 16;
```

Comment:
I'd consider doing `#![warn(unreachable_pub)]` and being more precise about what is meant here, that makes it a lot easier to understand the level of intended visibility -- that is, I think you really meant `pub(crate)` here?